zebrad/src/components/mempool/gossip.rs

```diff
@@ -41,7 +41,7 @@ where
         let txs_len = txs.len();
         let request = zn::Request::AdvertiseTransactionIds(txs);
 
-        info!(?request, "sending mempool transaction broadcast");
+        debug!(?request, "sending mempool transaction broadcast");
```

Comment:
I'd like to keep this log at info level, because it shows that the mempool is working.

This helps us diagnose mempool failures when we can't reproduce the error, or if the user can't activate debug logs.